Makefile

```diff
@@ -211,7 +209,7 @@ endif
 
 fmt:
 	@$(call print, "Formatting source.")
-	$(GOLIST) | $(XARGS) go fmt -x
+	gofmt -s -l -w .
```

Comment:
won't this format the vendor dir?

Reply:
yeah, but does it matter? 😛  I guess it could increase the duration...

Reply:
Updated to exclude the vendor dir!